meshio/_plot3d.py

```diff
@@ -0,0 +1,94 @@
+"""
+PLOT3D format is for structured grids only
+For more info, check out: PLOT3D User's Manual, chapter 8: Data File Formats
+Author: Samir OUCHENE
```

Comment:
We don't put author names in the files. Everyone can read up in the Git history who contributed what.

Reply:
Yes. Sorry, It was a mistake.